airbyte-ci/connectors/auto_merge/src/auto_merge/consts.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from __future__ import annotations
+
+AIRBYTE_REPO = "airbytehq/airbyte"
+AUTO_MERGE_LABEL = "auto-merge"
+BASE_BRANCH = "master"
+CONNECTOR_PATH_PREFIXES = {
+    "airbyte-integrations/connectors/",
```

Comment:
Nit: this one has trailing `/`, the other two do not. I don't think this is intentional?